src/main/java/ratpack/zipkin/internal/RatpackCurrentTraceContext.java

```diff
@@ -0,0 +1,41 @@
+package ratpack.zipkin.internal;
+
+import brave.internal.Nullable;
+import brave.propagation.CurrentTraceContext;
+import brave.propagation.Propagation;
+import brave.propagation.TraceContext;
+import org.slf4j.MDC;
+
+public final class RatpackCurrentTraceContext extends CurrentTraceContext {
+
+  private final MDCProxy mdc;
+
+  private final TraceContext.Extractor<MDCProxy> extractor = Propagation.B3_STRING.extractor(MDCProxy::get);
+  private final TraceContext.Injector<MDCProxy> injector = Propagation.B3_STRING.injector(MDCProxy::put);
```

Comment:
Log correlation would be a good reason to keep it :)  Easy to add back I suppose, still I seem to have a bug in here or Ratpack has a bug in their MDCInterceptor because it didn't seem to work.  I was probably just doing something stupid though ;)